# Define goal for trigger
bool cancel
---
# Define the result
bool success
string status
---
# Define a feedback message
string status
bool success
geometry_msgs/PoseArray poses

